app/javascript/packs/admin_reservation.js

```diff
@@ -0,0 +1,39 @@
+$(function(){
+  $('#admin-reservation').on('change','#bill_filter', function() {
+    let range = $(this).val();
+    let url = $(this).data('url');
+    $.ajax({
+      url: url,
+      method: 'GET',
+      data: {
+        range: range
+      }
+    });
+  });
+  $('#admin-reser').on('click', '.bill-cancle',function() {
+    let sure = confirm(I18n.t('users.show.sure'));
+    if (sure == true) {
```

Comment:
nếu true thì chỉ cần là sure, false thì !sure là được ko cần == true
```suggestion
    if (confirm(I18n.t 'users.show.sure')) {
```